.github/CHANGELOG.md

```diff
@@ -71,6 +25,39 @@
 
   See the `default.qubit.autograd` documentation for more details.
 
+<h4>New algorithms and templates</h4>
+
+* Added built-in QAOA functionality via the new `qml.qaoa` module.
+  [(#712)](https://github.com/PennyLaneAI/pennylane/pull/712)
+  [(#718)](https://github.com/PennyLaneAI/pennylane/pull/718)
+  [(#741)](https://github.com/PennyLaneAI/pennylane/pull/741)
+  [(#720)](https://github.com/PennyLaneAI/pennylane/pull/720)
+
+  ```pycon
+  >>> graph = nx.Graph([(0, 1), (1, 2)])
+  >>> cost_h, mixer_h = qml.qaoa.maxcut(graph)
+  ```
+
+  This includes the following features:
+
+  * New `qml.qaoa.x_mixer` and `qml.qaoa.xy_mixer` functions for defining Pauli-X and XY
+    mixer Hamiltonians.
+
+  * MaxCut: The `qml.qaoa.maxcut` function allows easy construction of the cost Hamiltonian
+    and recommended mixer Hamiltonian for solving the MaxCut problem for a supplied graph.
+
+  * Layers: `qml.qaoa.cost_layer` and `qml.qaoa.mixer_layer` take cost and mixer
+    Hamiltonians, respectively, and apply the corresponding QAOA cost and mixer layers
+    to the quantum circuit
+
+* Added an `ApproxTimeEvolution` template to the PennyLane templates module, which
+  can be used to implement Trotterized time-evolution under a Hamiltonian.
+  [(#710)](https://github.com/XanaduAI/pennylane/pull/710)
+
+* Added a `qml.layer` template-constructing function, which takes a unitary, and
+  repeatedly applies it on a set of wires to a given depth.
+  [(#723)](https://github.com/PennyLaneAI/pennylane/pull/723)
```

Comment:
Should add small example and/or diagram here!

Reply:
@josh146 Sorry, I forgot to address this point yesterday, what exactly did you have in mind for a diagram/example?

Reply:
Maybe just showing the circuit output when layering a basic sequence of gates?

Reply:
Yep, sound great!